app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -345,6 +362,12 @@ def single_item_search_builder
       deprecation_deprecate :single_item_search_builder
 
       def collection_params
+        # For backward compatibility, need to support params[:collection] for existing APIs
+        @collection_params = params[:collection]
+        @collection_params ||= params[:pcdm_collection]
+      end
+
+      def collection_params_without_permissions
         @participants = extract_old_style_permission_attributes(params[:collection])
```

Comment:
Do we still need to support passing in permissions in the params?  This was marked for deprecation starting at 3.0, but didn't get removed.  See related methods...  [controllers/hyrax/dashboard/collections_controller.rb](https://github.com/samvera/hyrax/blob/main/app/controllers/hyrax/dashboard/collections_controller.rb#L352-L376)

Reply:
I am not sure if this is connected, but Bulkrax might include the ability to create collections when a work is imported (created). Would that require passing in parameters here that aren't defined or visible on the form?

Reply:
seems like if we drop them we need to bump the major version.